sim/midas/src/main/scala/midas/core/FPGATop.scala

```diff
@@ -415,6 +415,8 @@ class FPGATopImp(outer: FPGATop)(implicit p: Parameters) extends LazyModuleImp(o
   def genHeader(sb: StringBuilder, target: String)(implicit p: Parameters) = {
     outer.genHeader(sb)
 
+    sb append "#ifdef GET_TARGET_CONFIG\n"
```

Comment:
I know this is has the target name in it, but a better name might be "METASIM_INTERFACE_CONFIG". I had a similar qualm with "TargetConfig" before.